### PR TITLE
add missing choices_as_values options

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

**Comment by xabbuh** [comment]
ping @HeahDude :)

File: form/dynamic_form_modification.rst [hunk 1]
```diff
@@ -490,9 +490,10 @@ sport like this::
                     $positions = null === $sport ? array() : $sport->getAvailablePositions();
 
                     $form->add('position', 'entity', array(
-                        'class'       => 'AppBundle:Position',
+                        'class' => 'AppBundle:Position',
```
Comment (HeahDude) [inline form/dynamic_form_modification.rst]: This is not related to your PR but what about dropping this doctrine notation in favor of class constants in the docs? The only advantage it has today IMO is eventually for config files with factory methods (even there I always prefer fqcn though).

**Review by HeahDude — APPROVED** [review]
👍

File: form/dynamic_form_modification.rst (hunk 1 above)
Comment (xabbuh) [inline form/dynamic_form_modification.rst]: I agree. In #7907 we did that when using the `getRepository()` method. I will prepare another PR for other places where we use this (like here).

**Pull request merged**